features/step_definitions/admin_manage_members_steps.rb

```diff
@@ -106,9 +106,9 @@ def find_remove_link_for_person(full_name)
 
 Then(/^I should be able to send a message to admin$/) do
   steps %Q{
-    When I fill in "What would you like to tell us?" with "I sad that I have been banned."
-    And I press "Send feedback"
-    Then I should see "Thanks a lot for your feedback!" within ".flash-notifications"
+    When I fill in "Your message to %{service_name} team" with "I sad that I have been banned."
```

Comment:
@thomasmalbaux, change `"Your message to %{service_name} team"` to `"feedback_content"`. That should fix the tests and avoid having to work with variables here.